packages/fxa-content-server/app/scripts/templates/sms_send.mustache

```diff
@@ -9,9 +9,14 @@
     </header>
     <section class="send-sms">
       <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Connect another device{{/t}}"></div>
-      <h1>
+      <h1 data-id="prompt-header">
         {{^userHasAttachedMobileDevice}}
```

Comment:
ISTM that we should only enroll a user into the experiment if they don't have a mobile device already connected?

Reply:
@vbudhram This is correct. If the user has a mobile device attached they should see `Still adding devices?`, otherwise there's an A/B test between these two versions of "phone" copy.

Reply:
Ah Gotcha, I would then probably want to limit the enrollment population to these users. Thinking something like

```javascript
let isInSmsSendHeaderExperiment = false;
if (this._userHasAttachedMobileDevice) {
    // Only enroll users that don't have a mobile device into this experiment
    const experimentGroup = this.getAndReportExperimentGroup('sendSmsHeader');
    isInSmsSendHeaderExperiment = experimentGroup === 'syncBrowser';
}
```